aa_engine/aas_addcontrast.m

```diff
@@ -262,6 +262,14 @@
     subjname = {aap.acq_details.subjects.subjname};
 end
 
+sub_id = 0;

```

Comment:
I guess we do not need this loop any more.

Reply:
Indeed. I'll remove it on the next commit.